docs/scripting.md

```diff
@@ -510,10 +510,10 @@ Using previous examples:
 
   robot.hear /midnight train/i, (res) ->
     robot.http("https://midnight-train")
-      .get() (err, res, body) ->
+      .get() (err, response, body) ->
         if err
           res.reply "Had problems taking the midnight train"
-          robot.emit 'error', err, res
+          robot.emit 'error', err, response
```

Comment:
This one should be `res`. The error handler can use the `Response` object to reply back to the user that something happened.